### PR TITLE
MAINT: changed needed after renaming `master` branch to `main`

### PR DESCRIPTION
**Pull request by jarrodmillman**
This is for after @rgommers renames the master branch main.

### CODE REVIEW

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
         # this is taken verbatim from the numpy azure pipeline setup.
         set -xe
         # same version of gfortran as the open-libs and numpy-wheel builds
-        curl -L https://github.com/MacPython/gfortran-install/raw/master/archives/gfortran-4.9.0-Mavericks.dmg -o gfortran.dmg
+        curl -L https://github.com/MacPython/gfortran-install/raw/main/archives/gfortran-4.9.0-Mavericks.dmg -o gfortran.dmg
```
Comment (matthew-brett) [inline .github/workflows/macos.yml:39]: I think this repo is still using `master`.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -300,7 +300,7 @@ stages:
     # are the primary providers of the SciPy wheels available
     # on PyPI, so we now regularly test that the version of
     # _distributor_init.py in our wheels repo is capable of
-    # loading the DLLs from a master branch wheel build
+    # loading the DLLs from a main branch wheel build
```
Comment (matthew-brett) [inline azure-pipelines.yml:303]: This refers to `scipy-wheels`?  Is that `master` or `main`?

**Review by matthew-brett — COMMENTED** [review]
I think there are a couple of references to other repos still using `master`.

File: azure-pipelines.yml (hunk 2 above)
Comment (jarrodmillman) [reply azure-pipelines.yml:303]: It looks like it is still master: https://github.com/MacPython/scipy-wheels

**Comment by rgommers** [comment]
Can you take this out of Draft @jarrodmillman? I'd like to be able to merge it right after the rename.

I'll do `.gitmodules` separately, please don't touch it in this PR. It requires a bit more gymnastics to not break things I think.

**Comment by rgommers** [comment]
> I'll do `.gitmodules` separately, please don't touch it in this PR. It requires a bit more gymnastics to not break things I think.

After some checking: I think this'll work without any issues, branch name is not included. Let me go try and see if CI or local updating breaks.

**Comment by rgommers** [comment]
> I think this'll work without any issues, branch name is not included. Let me go try and see if CI or local updating breaks.

Tried with the `unuran` repo - if there's no complaints then let's do all the other ones tomorrow.

**Comment by rgommers** [comment]
CircleCI may have a hiccup: https://support.circleci.com/hc/en-us/articles/360040969693-VCS-Default-Branch-Isn-t-Reflected-on-CircleCI. This just happened on the `scipy-sphinx-theme` repo. The article claims it will resolve itself. If not, removing and re-adding the CircleCI webhook should fix it.

**Comment by rgommers** [comment]
CircleCI is indeed annoying - it didn't unbreak itself. This is a bit of a pain with our auto-deployed html docs.

**Review by rgommers — APPROVED** [review]
Thanks @jarrodmillman. And thanks for having a look @matthew-brett. There were a lot more instances of `master`, I pushed an update. Should be good to go now.

**Review by larsoner — APPROVED** [review]
LGTM +1 for merge. Looks like other PRs I've made with similar CI infrastructure, so hopefully it works (or is close so another quick PR will fix any remaining errors). @rgommers feel free to click the green button if you agree!

**Comment by rgommers** [comment]
Okay, there we go `:fingers-crossed:`

File: .circleci/config.yml [hunk 3]
```diff
@@ -30,7 +30,7 @@ jobs:
               echo "Merging $(cat merge.txt)";
               git remote add upstream https://github.com/scipy/scipy.git;
               git pull --ff-only upstream "refs/pull/$(cat merge.txt)/merge";
-              git fetch upstream master;
+              git fetch upstream main;
```
Comment (tylerjereddy) [inline .circleci/config.yml:33]: of course, for backports PRs this wasn't correct before or after per gh-15425; I'm somewhat inclined to just adopt this quick fix for now and do the proper backport-sensitive fix later though

**Pull request merged**